ansible/kubernetes/configure_kubernetes.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Set SSH settings
```

Comment:
Update playbook name. This is for managing swap, /etc/hosts, and quality of life stuff.